libpysal/weights/util.py

```diff
@@ -211,6 +207,82 @@ def lat2W(nrows=5, ncols=5, rook=True, id_type='int', **kwargs):
     return W(w, weights, ids=ids, id_order=ids[:], **kwargs)
 
 
+def xrast2W(rasterf, rook=True, **kwargs):
```

Comment:
I'd replace `rasterf` for `da`, which is the convention I've seen in xarray.

Also, @slumnitz suggested to stick to the same API for the `rook` as in `lat2W`

Reply:
it follows same API which exist in [`lat2W`](https://github.com/pysal/libpysal/blob/16a7f585bb95012bdd3dc9267bf48065f2afd1fa/libpysal/weights/util.py#L128)... 
since we've shifted to different method we don't need to pass `rook` argument anymore